src/dbus/server/introspect.xml

```diff
@@ -246,8 +246,15 @@
       <arg name="epskc" type="s" direction="out"/>
     </method>
 
-    <!-- DeactivateEphemeralKeyMode: Deactivate ePSKc mode. -->
+    <!-- DeactivateEphemeralKeyMode: Deactivate ePSKc mode.
+      @retain_active_session: 
+        "false" - Default value. Disconnects the Border Agent from any active secure sessions.
```

Comment:
Why "Default value" here? I think line 2017 in src/dbus/server/dbus_thread_object_rcp.cpp is making the "retain_active_session" a mandatory argument. Given it is not an optional value, there is no value to mention about the default value because the client always need to provide the argument explicitly.

Reply:
Thanks, changed the description to map the latest code.